naga/src/front/wgsl/error.rs

```diff
@@ -1032,7 +1029,12 @@ impl<'a> Error<'a> {
                         kind.tracking_issue_num()
                     )]
                 } else {
-                    vec![]
+                    vec![
+                        format!(
+                            "You can enable this extension by adding `enable {};` at the top of the shader.",
```

Comment:
issue: "at the top of the shader" is a good informal abbreviation in general for what is actually meant, but for a new user that may try to put this, say, under their small `const MY_THING = 1;` on line 1 of their WGSL file, it's important to be precise in this hint.

suggestion: "You can enable this by adding `enable {};` before global items in the file."